02-clients/01-studio.md

```diff
@@ -0,0 +1,424 @@
+   query to show the results of the query execution attempt.
+
+   ![Write the Schema](../images/studio/project-schema-query-run.png)
+
+4. If it's the `write` transaction type — wait for transaction control options to be ready and commit or close the 
+   transaction (by clicking on the [**Commit transaction**] (✔) or [**Close transaction**] (x) buttons respectively).
+
+The result of the query is displayed in the [**Run**] panel under the Text editor.
+
+If any error occurs during the query validation or transaction it is shown as red popup in the bottom right corner of 
+the TypeDB Studio window. Additional information about the error is displayed in the results of the query in the 
+[**Run**] panel.
+
+If a `write` transaction committed successfully — a blue pop-up with a transaction confirmation is displayed. See 
+example below.
+
+![Commit the Transaction](../images/studio/project-schema-committed.png)
+
+## Types explorer capabilities
+
+Under the [**Projects**] panel that shows the project directory TypeDB has [**Types**] panel that shows all types of 
+the schema of the current database. 
+
+<div class="note">
+[Note]
+If the [**Types**] panel is not displayed — click the [**Types**] vertical tab on the left edge of the TypeDB 
+Studio window.
+</div>
+
+All types are divided in three top level [base types](../0001-typedb/02-dev/02-schema.md#types):
+
+- Attribute
+- Entity
+- Relation
+
+In an empty database without a schema, only these three base types exist.
+
+In a database with a schema all types displayed in the exact hierarchy they were created (subtypes inside their 
+supertypes). We can easily collapse or expand all the types in the structure tree by clicking the associated buttons 
+on the top of the [**Types**] panel with double chevrons (arrow heads) pointing down or up respectively.
+
+![Types explorer](../images/studio/types-explorer.png)
+
+Double-clicking on any type will open the [**[Type browser](#type-browser)**] with information about the selected type 
+in a tab of the [**Text editor**].
+
+[**Types**] panel can be closed by clicking the **X** icon in the top right corner of the panel. To reopen the panel 
+click the [**Types**] tab on the left edge of the TypeDB Studio window.
+
+### Export schema
+
+We can export schema of the database by clicking the [**Export Schema Types**] button (second in row from left to 
+right) on the top of the [**Types**] panel. It creates a new tab with inserted TypeQL code that creates the exact 
+schema that is in the current database.
+
+<div class="note">
+[Warning]
+The [**Export Schema Types**] button doesn't include rules into its output. That will be fixed in some of the next 
+versions of the TypeDB Studio. 
+</div>
+
+### Type browser
+
+By double-clicking any type we can open the type browser window that shows all information about the selected type and
+allows us to easily edit the type. 
+
+![Types browser](../images/studio/types-browser.png)
```

Comment:
This is a type page! The Type Browser is, as above, the panel on which all the types are listed in Studio's main view.